packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/User.kt

```diff
@@ -68,6 +68,34 @@ public interface User {
      */
     public val identities: List<UserIdentity>
 
+    /**
+     * Returns the provider type used to log the user
```

Comment:
This seems to cut off.

Maybe "Returns the provider type used to log the user in."

Reply:
Also, we should document the behaviour when the user logs out.